docs/Angular-tutorial.md

```diff
@@ -0,0 +1,67 @@
+# Get started with UI5 Web Components & Angular
+
+In this tutorial, we are going to show how to get add UI5 Web Components in your application. UI5 Web Components can be added to both new Angular application as well as already existing one.
```

Comment:
In this tutorial we are going to show how to add UI5 Web Components to your application. UI5 Web Components can be added to both new Angular applications, as well as already existing ones.